src/rules/Amazon/ScrapAmazon.ts

```diff
@@ -1,20 +1,27 @@
-import { TYPE_AMAZON } from '../index';
```

Comment:
Single quote is prefered character for imports

Reply:
Not according to tslint, but you are the maintainer so if you want single quotes it will be single quotes.